ios/FluentUI/Vnext/List/ListCell.swift

```diff
@@ -216,3 +225,53 @@ struct ListCellButtonStyle: ButtonStyle {
             .background(configuration.isPressed ? Color(tokens.highlightedBackgroundColor) : Color(tokens.backgroundColor))
     }
 }
+
+@objc public class MSFPersonaCellState: NSObject, ObservableObject, PersonaCellProtocol {
+    @objc @Published public var leadingView: UIView?
+    @objc @Published public var title: String = ""
+    @objc @Published public var subtitle: String = ""
+    @objc @Published public var titleTrailingAccessoryView: UIView?
+    @objc @Published public var subtitleTrailingAccessoryView: UIView?
+}
+/// UIKit wrapper that exposes the SwiftUI Persona Cell implementation
+@objc open class MSFListPersona: NSObject, FluentUIWindowProvider {
```

Comment:
I thought of the the persona cell to be a new subspec that depends on the cell subspec.
Do you think we can try to detach it into a new folder and make sure that the dependencies are correctly defined (Persona Cell depends on List Cell but List Cell does not depend on Persona Cell)?